### ISSUE TITLE
Logging a string is disabled at warn and error levels in LocationIgnorantKLogger

### ISSUE THREAD

**Issue by dondod** [MicroUtils/kotlin-logging#276]
https://github.com/oshai/kotlin-logging/blob/master/src/jvmMain/kotlin/io/github/oshai/slf4j/internal/LocationIgnorantKLogger.kt#L240

This doesn't seem right (checking for `trace` instead of `warn`):

```kotlin
  override fun warn(msg: String?) {
    if (isTraceEnabled) {
      underlyingLogger.warn(msg)
    }
  }
```

same for `error`: 
```kotlin
 override fun error(msg: String?) {
    if (isTraceEnabled) {
      underlyingLogger.error(msg)
    }
  }
```

**Comment by oshai** [comment]
Thanks for reporting!

**Comment by oshai** [comment]
Fixed in 4.0.0-beta-18 (I missed one spot so created to releases for it)

**Comment by dondod** [comment]
Thanks!  Looks like the same issue exists with `debug` and `info` though:
```kotlin
 override fun debug(msg: String?) {
    if (isTraceEnabled) {
      underlyingLogger.debug(msg)
    }
  }
```

```kotlin
override fun info(msg: String?) {
    if (isTraceEnabled) {
      underlyingLogger.info(msg)
    }
  }
```

**Comment by oshai** [comment]
Thanks! fixed in 4.0.0-beta-19.
